cms/test_utils/project/templates/registration/logged_out.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+
+{% block content %}
+
+<p>{% trans "Thanks for spending some quality time with the Web site today." %}</p>
```

Comment:
Some may see this as "passive aggressive". "Web site" should probably just be "website".

Perhaps we should just say something to confirm the action: "You have successfully logged-out." perhaps?